### PR TITLE
EPMRPP-64927: change field sender name

### PR DESCRIPTION
**Pull request by oranje322**

### CODE REVIEW

**Comment by chivekrodis** [comment]
Please recheck task, looks like you change the wrong field. Or update task name correctly

**Review by chivekrodis — CHANGES_REQUESTED** [review]
Please recheck task, looks like you change the wrong field. Or update task name correctly

File: app/localization/translated/uk.json [hunk 1]
```diff
@@ -491,7 +491,7 @@
   "EmailFormFields.portFieldHint": "Можливі тільки цифри від '1' до '65535'",
   "EmailFormFields.portLabel": "Порт",
   "EmailFormFields.protocolLabel": "Протокол",
-  "EmailFormFields.usernameLabel": "Ім’я користувача",
+  "EmailFormFields.usernameLabel": "Електронної пошти відправника",
```
Comment (chivekrodis) [inline app/localization/translated/uk.json:494]: Maybe better 'Електронна пошта відправника'

File: app/localization/translated/be.json [hunk 2]
```diff
@@ -491,7 +491,7 @@
   "EmailFormFields.portFieldHint": "Магчымы толькі лічбы ад '1' да '65535'",
   "EmailFormFields.portLabel": "Порт",
   "EmailFormFields.protocolLabel": "Пратакол",
-  "EmailFormFields.usernameLabel": "Імя карыстальніка",
+  "EmailFormFields.usernameLabel": "Адпраўшчык электроннай пошты",
```
Comment (chivekrodis) [inline app/localization/translated/be.json:494]: Maybe better 'Электронная пошта адпраўніка'

**Review by chivekrodis — CHANGES_REQUESTED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**